app/javascript/react/screens/App/Plan/Plan.js

```diff
@@ -171,29 +178,50 @@ class Plan extends React.Component {
             planRequestPreviouslyFetched &&
             !isRejectedPlanRequest &&
             planRequestTasksMutable.length > 0 && (
-              <PlanRequestDetailList
-                plan={plan}
-                planFinished={planFinished}
-                planRequestTasks={planRequestTasksMutable}
-                downloadLogAction={downloadLogAction}
-                downloadLogInProgressTaskIds={downloadLogInProgressTaskIds}
-                fetchAnsiblePlaybookTemplateAction={fetchAnsiblePlaybookTemplateAction}
-                fetchAnsiblePlaybookTemplateUrl={fetchPlanUrl}
-                ansiblePlaybookTemplate={ansiblePlaybookTemplate}
-                fetchOrchestrationStackUrl={fetchOrchestrationStackUrl}
-                fetchOrchestrationStackAction={fetchOrchestrationStackAction}
-                orchestrationStack={orchestrationStack}
-                cancelPlanRequestTasksUrl={cancelPlanRequestTasksUrl}
-                cancelPlanRequestTasksAction={cancelPlanRequestTasksAction}
-                selectedTasksForCancel={selectedTasksForCancel}
-                updateSelectedTasksForCancelAction={updateSelectedTasksForCancelAction}
-                deleteAllSelectedTasksForCancelAction={deleteAllSelectedTasksForCancelAction}
-                markedForCancellation={markedForCancellation}
-                failedMigrations={failedMigrations}
-                successfulMigrations={successfulMigrations}
-                notificationsSentList={notificationsSentList}
-                dispatchVMTasksCompletionNotificationAction={dispatchVMTasksCompletionNotificationAction}
-              />
+              <ListViewToolbar
+                filterTypes={planFinished ? FINISHED_PLAN_FILTER_TYPES : ACTIVE_PLAN_FILTER_TYPES}
+                sortFields={planFinished ? FINISHED_PLAN_SORT_FIELDS : ACTIVE_PLAN_SORT_FIELDS}
+                listItems={planRequestTasksMutable}
```

Comment:
The default sort type for `PlanRequestDetailList` should be `name`

Reply:
Ah, good catch